Web/starter_project_vanguard/data/db.json

```diff
@@ -0,0 +1,50 @@
+{
+  "social-projects": [
+    {
+      "image": "projects/hakimhub.svg",
+      "title": "Hakim Hub",
+      "content": "HakimHub is a platform that provides information about healthcare facilities and healthcare professionals in Ethiopia. Hakimhub makes information about hospitals, medical laboratories, and doctors conveniently accessible to its users.",
+      "leftAligned": false,
+      "isImageLeft": true
+    },
+    {
+      "image": "projects/tracksys.svg",
```

Comment:
same here only store the image file name

Reply:
I found it appropriate to store this way, as I have parent path /about before it